control/installation.xml

```diff
@@ -29,6 +29,27 @@
     <default_ntp_setup config:type="boolean">true</default_ntp_setup>
   </globals>
 
+  <software>
+    <clone_install_recommended_default config:type="boolean">true</clone_install_recommended_default>
+    <minimalistic_libzypp_config config:type="boolean">true</minimalistic_libzypp_config>
+    <display_support_status config:type="boolean">true</display_support_status>
+    <default_patterns>base kvm_server kvm_tools</default_patterns>
+    <!-- the cdata trick produces an empty string in the data
+	 instead of omitting the key entirely -->
+    <optional_default_patterns><![CDATA[]]></optional_default_patterns>
+  </software>
+  <partitioning>
+    <proposal_settings_editable config:type="boolean">true</proposal_settings_editable>
+    <try_separate_home config:type="boolean">false</try_separate_home>
+    <home_path>/var/lib/libvirt/</home_path>
+    <home_fs>xfs</home_fs>
+    <proposal_lvm config:type="boolean">false</proposal_lvm>
+    <root_space_percent>25</root_space_percent>
+    <root_base_size>15GB</root_base_size>
+    <root_max_size>30GB</root_max_size>
+    <proposal_lvm config:type="boolean">false</proposal_lvm>
+  </partitioning>
```

Comment:
problem here is that it is enabled always and not only for this role. I already extend definition  of system role, so it now can have globals, software and also partitioning. So you should move it back to system_role, otherwise it is applied always and not just when you select role.